docs/integrations/sources/jina-ai.md

```diff
@@ -0,0 +1,51 @@
+# Jina AI
```

Comment:
I think the title should probably have "Reader" in the name since (I believe) both the Search and Reader endpoints are advertised under "Reader" brand. (Also because Jina has several other AI services we are not trying to encapsulate here.)

```suggestion
# Jina AI Reader
```

Reply:
:white_check_mark: